app/code/community/Bolt/Boltpay/Model/BoltOrder.php

```diff
@@ -365,6 +446,17 @@ protected function getCorrectedTotal($projectedTotal, $magentoDerivedCartData)
         return $magentoDerivedCartData;
     }
 
+
+    /**
+     * Provides and opportunity to override Bolt shipping address data in custom plugin code
+     *
+     * @param array                  $shippingAddress    The shipping address in Bolt Array format
+     * @param Mage_Sales_Model_Quote $quote              the quote associated with the order
+     */
+    public function correctShippingAddress(&$shippingAddress, $quote) {
```

Comment:
Do you still need this? Hoping we can migrate everything to event if possible

Reply:
@daisy1754, will update